mwclient/client.py

```diff
@@ -1039,12 +1044,24 @@ def upload(
             filekey: Key that identifies a previous upload that was stashed temporarily.
             comment: Upload comment. Also used as the initial page text for new files if
                      `description` is not specified.
+            asynchronous: Whether the server should upload the file asynchronously.
+                            Must be used with the filekey of a previously stashed file
+            stash: If set, the file will be stashed instead of uploaded right away.
 
         Example:
 
             >>> client.upload(open('somefile', 'rb'), filename='somefile.jpg',
                               description='Some description')
 
+        Async Example:
+            >>> response = client.upload(open('somefile','rb'),
+                                        filename='somefile.jpg'
```

Comment:
There's a missing comma in this example:


```suggestion
                                        filename='somefile.jpg',
```